mpas_analysis/shared/interpolation/test_interpolate.py

```diff
@@ -0,0 +1,109 @@
+"""
+Created on Mon Oct 24 14:13:27 2016
+
+author: Luke Van Roekel
+date: 10-24-2016
+"""
+
+import numpy as np
+from interpolate import interp_fields, init_tree, lon_lat_to_cartesian
+    
+class TestClass:
+    def test_lat_to_cartesian(self):
+        lat_input = np.deg2rad(np.array([-90, 90, 0, 0, 0, 0, 45, 45, 45, 45]))
+        lon_input = np.deg2rad(np.array([0, 0, 0, 90, 180, 270, 0, 90, 180, 270]))
+        
+        x_input = np.cos(lat_input) * np.cos(lon_input)
+        y_input = np.cos(lat_input) * np.sin(lon_input)
+        z_input = np.sin(lat_input)
+        
+        x, y, z = lon_lat_to_cartesian(lon_input, lat_input, R=1.0)
+        
+        assert np.all(x == x_input)
```

Comment:
Please inherit the testclass in line 11:
```class TestInterp(TestCase):```

and then for asserts:

```self.assertEqual(x, x_input)```

This will let us have greater control of equals checking in the future.

Does this make sense?